types/styled-components/index.d.ts

```diff
@@ -145,34 +162,38 @@ export interface StyledComponent<
     ): StyledComponent<WithC, T, O, A>;
 }
 
-export interface ThemedStyledFunction<
+export type ThemedStyledFunctionBase<
     C extends keyof JSX.IntrinsicElements | React.ComponentType<any>,
     T extends object,
     O extends object = {},
     A extends keyof any = never
-> {
-    (
-        strings: TemplateStringsArray,
-        ...interpolations: Array<
+> =
+    // at least the first argument is required, whatever it is
+    <U extends object>(
+        first: NonNullable<
```

Comment:
Oh, this doesn't break usage as a template string because both `Interpolation` and `SimpleInterpolation` accept `string[] & { raw: string[] }` by being assignable to `ReadonlyArray<string>`.